apps/oxlint/test/fixtures/context_properties/output.snap.md

```diff
@@ -3,6 +3,12 @@
 
 # stdout
 ```
+  x context-plugin(log-context): cwd:
```

Comment:
this is super tricky to test because of our snapshot architecture ect.

Let's include an assertion inside `context_properties.plugin.ts` to assert that the len is non-zero, and it's the same as `process.cwd()` ?